app/assets/stylesheets/admin/questions/_questions.scss

```diff
@@ -1,3 +1,22 @@
+.now-editing {
+  color: $red2;
```

Comment:
I think I need to update these $red2s to actually be a division custom color

Reply:
There are probably a few other places that's a thing, if this is an easy change to do now go ahead, otherwise I think we can take it on during a general pass to find these sorts of inconsistencies